@xen-orchestra/web/src/components/site/dashboard/Alarms.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <UiCard class="site-dashboard-alarms">
+    <UiCardTitle>
+      {{ t('alarms') }}
+      <UiCounter
+        v-if="alarms.length !== 0 && areAlarmsReady"
+        accent="danger"
+        size="small"
+        variant="primary"
+        :value="alarms.length"
+      />
+    </UiCardTitle>
+    <VtsLoadingHero v-if="!areAlarmsReady" type="card" />
+    <VtsAllGoodHero v-else-if="alarms.length === 0" type="card" />
+    <div v-else class="alarm-items">
```

Comment:
`alarm-list-container` would probably be a better name here

```suggestion
    <div v-else class="alarm-list-container">
```